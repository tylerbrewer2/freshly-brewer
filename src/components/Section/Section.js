import React from 'react';
import styled from 'styled-components';

// Style Helpers
import MediaWidths from '../../styles/helpers/MediaWidths';

const SectionCore = ({className, children }) => {
  return (
    <section className={className}>
      {children}
    </section>
  )
}

const Section = styled(SectionCore)`
  padding: 120px 0;
  text-align: ${props => props.centered ? 'center' : ''};
  max-width: ${props => props.full ? '' : '90%'};
  margin: 0 auto;

  @media (${MediaWidths.mobile}) {
    padding: 60px 0;
  }
`

export default Section;