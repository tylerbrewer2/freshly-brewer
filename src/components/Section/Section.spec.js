import React from 'react';
import renderer from 'react-test-renderer';

import Section from './Section';

it('renders correctly', () => {
  const tree = renderer.create(
    <Section>
      Test section
    </Section>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
