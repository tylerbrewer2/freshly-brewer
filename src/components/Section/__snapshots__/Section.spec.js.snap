exports[`test renders correctly 1`] = `
<section
  className="sc-bdVaJa cYbOri">
  Test section
</section>
`;
